mozetl/taar/taar_lite_guidguid.py

```diff
@@ -241,8 +187,26 @@ def main(date, bucket, prefix):
             value_json[_id] = n
         result_json[key_addon] = value_json
```

Comment:
Is this missing the return value?

Reply:
I like structuring these jobs in the following way:
```
extract :: IO -> DataFrame
transform :: DataFrame -> DataFrame
load :: DataFrame -> IO
```

The main advantage is having a consistent container type for testing the transformation method, which should do the bulk of the work. Additionally, IO can be mocked easily in this repository through the use of the boto3 `@mock_s3` decorator, which redirects writes to the filesystem.